### PR TITLE
CU-969v_boom_Jerry-Krusinski

### PR DESCRIPTION
**Pull request by lanakrusinski**
Make me a pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # clickup_test
 github api test
+
+Small change
```
Comment (lanakrusinski) [inline README.md:4]: please change this

Comment (jkrusinski) [inline README.md:4]: please change this

**Comment by lanakrusinski** [comment]
a comment there

File: README.md (hunk 1 above)
Comment (jkrusinski) [inline README.md:4]: this is also a problem

**Pull request merged**